x-pack/plugins/enterprise_search/server/routes/workplace_search/sources.ts

```diff
@@ -241,9 +241,6 @@ export function registerAccountSourceReauthPrepareRoute({
         params: schema.object({
           id: schema.string(),
         }),
-        query: schema.object({
-          kibana_host: schema.string(),
-        }),
       },
```

Comment:
It looks like `query` is still being sent as an empty object above. We either need to leave `query` here or stop sending it in the logic file above, right?